.github/tests/setup.sh

```diff
@@ -28,6 +28,11 @@ setup_cypht() {
         sed -i 's/mysql/sqlite/' tests/phpunit/mocks.php
         sed -i "s/'host'/'socket'/" tests/phpunit/mocks.php
     fi
+
```

Comment:
No need for these sed below as again as USER_CONFIG_TYPE is set to DB

Reply:
Even with USER_CONFIG_TYPE=DB we still need them, as you can see with this screenshort i tried to remove them to show you the result.
<img width="1384" alt="Screenshot 2024-08-29 at 01 58 15" src="https://github.com/user-attachments/assets/31fff12c-19a5-44eb-9042-c046815f8a92">

Reply:
I removed all and keep ATTACHMENT_DIR only.